src/edgepi/pwm/edgepi_pwm.py

```diff
@@ -26,6 +33,13 @@ def __init__(self, pwm_num: GpioPins):
         self.chip = self.__pwm_pin_to_channel[pwm_num].value.chip
         self.pwm.open_pwm()
 
+    def __check_range(self, target, range_min, range_max) -> bool:
+        """Validates target is in range between a min and max value"""
+        if range_min <= target <= range_max:
+            return True
+
+        raise ValueError(f"Target {target} is out of range ")
```

Comment:
could you include the range here in the error message? users will appreciate it : D

Reply:
ad932314ef22a561d9dcf868a506e860dec2d76a